qucs/qucs/mouseactions.cpp

```diff
@@ -319,6 +356,8 @@ void MouseActions::MMoveElement(Schematic *Doc, QMouseEvent *Event)
  */
 void MouseActions::MMoveWire2(Schematic *Doc, QMouseEvent *Event)
 {
+//ALYS - i saved old code inside #if 0 block
```

Comment:
a branch is merely a pointer to a commit. your PR lives as long as you dont delete the branch.

you should make yourself familiar with the `git reset` and `git rebase` commands, as well as with forced (non-forward) push. git might be a bit confusing for starters, but all of this makes perfect sense (promised).